docs/src/transforms.md

```diff
@@ -0,0 +1,89 @@
+# Tranforms
+
+## Column selection
```

Comment:
I think we can simply use a section per transform to avoid this naming issue. It is cumbersome to type it all manually, but if you use a fancy editor like VIM or Emacs you can automate this process with a macro. Or write a Julia program that spits out the text of this documentation page.

Reply:
We could have used documenter.jl autodocs features but it doesn`t create a clickable index as far as I know. If you find something in their docs that allows us to automatically add all docstrings of transforms and get links for free in the tab, that would be great.

Reply:
In this case, will the name of the section be the name of the transformation?

Reply:
Yes, too many sections but I think it is a reasonable solution for the clickable docs.